input.go

```diff
@@ -234,6 +234,15 @@ func IsStandardGamepadLayoutAvailable(id GamepadID) bool {
 	return uiDriver().Input().IsStandardGamepadLayoutAvailable(id)
 }
 
+// UpdateStandardGamepadLayoutMappings updates the gamepad layout definitions using a set of definitions in SDL_GameControllerDB format.
+//
+// This functionality is only supported on desktop systems and will just return an error elsewhere.
+//
+// UpdateStandardGamepadLayoutMappings must be called on the main thread before the main loop.
+func UpdateStandardGamepadLayoutMappings(mappings string) error {
+	return uiDriver().Input().UpdateStandardGamepadLayoutMappings(mappings)
```

Comment:
Just returning a boolean value should be fine. As some platforms (js) don't support it on purpose, this should not be an 'error'.